phpBB/includes/functions_user.php

```diff
@@ -288,6 +289,12 @@ function user_add($user_row, $cp_data = false, $notifications_data = null)
 
 	$user_id = $db->sql_nextid();
 
+	//	Check users_first_characters
+	$users_first_characters = $cache->get('users_first_characters');
+	if (($users_first_characters !== false) && (! array_key_exists(mb_substr($username_clean, 0, 1), $users_first_characters)))
+	{
+		$cache->destroy('users_first_characters');
```

Comment:
Instead of adding another global variable for the cache, use the container since it's available, so:
```php
$phpbb_container->get('cache')->destroy('users_first_characters');
```

Reply:
Then just do `$cache = $phpbb_container->get('cache')` instead of `global $cache`

Reply:
build failed
phpbb_console_user_add_test::test_add_no_dialog
Exception: Could not find service: cache

Reply:
try getting `cache.driver` instead of `cache`